test/test-utils.sh

```diff
@@ -397,12 +397,10 @@ function make_random_string() {
     else
         local END_POS=8
     fi
-    if [ "$(uname)" = "Darwin" ]; then
-        local BASE64_OPT="--break=0"
-    else
-        local BASE64_OPT="--wrap=0"
-    fi
-    "${BASE64_BIN}" "${BASE64_OPT}" < /dev/urandom 2>/dev/null | tr -d /+ | head -c "${END_POS}"
+    local READ_SIZE=$((END_POS * 2))
+    local BASE64_OPT="--wrap=0"
+
+    head -c "${READ_SIZE}" < /dev/urandom | "${BASE64_BIN}" "${BASE64_OPT}" | tr -d '/+' | head -c "${END_POS}"
```

Comment:
I believe that the return string just needs to be N characters that are in the base64 character set so we can simplify this without base64 via:
```
tr -cd A-Za-z0-9 < /dev/urandom | head -c ${END_POS}
```
I don't think the initial `head -c ${READ_SIZE}` is wise since this does not guarantee that any valid characters will be returned.

Reply:
Thank you.
Indeed, there's no need to use base64.
By the way, an error occurred if `LC_TYPE=C` was not specified when executing the `tr` command.
I will mark this comment as resolved and re-upload the corrected code.
Please review again.